<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shailesh" failures="1" tests="9" name="Test" time="8.685" errors="0" timestamp="2024-12-14T16:01:24 IST">
  <testcase classname="testngTutorial.testngTutorial.Day2" name="Demo1" time="1.987"/>
  <testcase classname="demo.Day3" name="demo3" time="0.009"/>
  <testcase classname="testngTutorial.testngTutorial.Day1" name="sumsung" time="0.007"/>
  <testcase classname="testngTutorial.testngTutorial.Day2" name="Demo2" time="0.051">
    <failure type="java.lang.AssertionError" message="expected [ed] but found [dd]">
      <![CDATA[java.lang.AssertionError: expected [ed] but found [dd]
at testngTutorial.testngTutorial.Day2.Demo2(Day2.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- Demo2 -->
  <testcase classname="demo.Day3" name="demo4" time="0.0"/>
  <testcase classname="testngTutorial.testngTutorial.Day1" name="splender" time="0.001"/>
  <testcase classname="testngTutorial.testngTutorial.Day1" name="splender" time="0.002"/>
  <testcase classname="testngTutorial.testngTutorial.Day2" name="Demo3" time="6.316"/>
  <testcase classname="testngTutorial.testngTutorial.Day1" name="iphone" time="0.0"/>
</testsuite> <!-- Test -->
